src/System.DirectoryServices/tests/Configurations.props

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-        netstandard-Windows_NT;
+        netcoreapp-Windows_NT;
```

Comment:
curious, why not netstandard-Windows_NT ?

Reply:
They can't run in uap :) if I leave netstandard they will be running there also.

Reply:
Why did you go this route with tests but not the src?

Reply:
The source is built against netstandard and uap, when built against uap we produce the PlatformNotSupported assembly as we want this to netstandard compatible and the identity with desktop is the same so we don't need a façade. 

Do you prefer going this route in the src also?